TA-cyences/metadata/default.meta

```diff
@@ -31,6 +31,15 @@ export = system
 export = system
 
 
+### ALERT ACTIONS
+
+[alert_actions]
+export = system
+
+[alerts]
```

Comment:
Why alerts?

Reply:
Alert_actions to show the alerts action and alerts to show the alert form/html. (without alerts it won't show any fields under the alert action)